middleware/router-auth.ts

```diff
@@ -3,12 +3,12 @@ import type {Route} from 'vue-router'
 
 export default function({store, redirect, route} : Context)
 {
-    if(route.name == null) redirect ('/')
-    if(store.state.auth.user != null && (route?.name?.split('/').some(record => record == 'auth')))
+    // if(route.name == null) redirect ('/')
+    if(store.state.auth.user != null && (route.name == null || route?.name?.split('/').some(record => record == 'auth')))
     {
         redirect('/home')
     }
-    if(store.state.auth.user == null && (isAdminRoute(route))) {
+    if(store.state.auth.user == null && (route.name == null || isAdminRoute(route))) {
```

Comment:
This is all great code, but i'm not sure if its best to redirect them to login if they aren't logged in. For new users, you might want them to explore the homepage and about quento before they sign in, and for old users they can just quickly click the sign in button on the top.